database/factories/UserFactory.php

```diff
@@ -1,29 +0,0 @@
-<?php
```

Comment:
Why remove it instead of updating it to new conditions? With factories inside of tests you can generate a set amount of models randomly, which could be great for our unit-tests.

Reply:
It's dead code currently and it would be necessary to introduce it for all models as well as seeder classes. Then re-write the test code. In the end we wouldn't benefit as development is going to be paused again after store release.